transformer_engine/debug/features/utils/stats_computation.py

```diff
@@ -199,6 +199,12 @@ def _get(buffers, stat_name):
     ),
 }
 
+FP8_NEGATIVE_ZERO = 128  # represnts -0.0 in fp8
+zero_values = {
+    device: torch.tensor([0, FP8_NEGATIVE_ZERO], device=device)
+    for device in [torch.device(f"cuda:{i}") for i in range(torch.cuda.device_count())]
+}
```

Comment:
I'm worried that it could be a bad idea for every rank to touch every local GPU at import time. At the very least this will incur some unnecessary initialization.

How about we add a helper function that resembles [`torch.count_nonzero`](https://docs.pytorch.org/docs/stable/generated/torch.count_nonzero.html):
```python
def count_nonzero_fp8(fp8_data: torch.Tensor) -> torch.Tensor:
    fp8_data = fp8_data.view(dtype=torch.uint8)
    zero_vals = torch.tensor([0, 128], device=fp8_data.device, dtype=torch.uint8)
    return fp8_data.numel() - torch.isin(fp8_data, zero_vals).sum()
```
Calling it would look something like:
```python
lambda x, aux_dict: (
    x.count_nonzero()
    - count_nonzero_fp8(aux_dict...)
)
```